projects/microphysics/argo/argo.yaml

```diff
@@ -150,7 +146,7 @@ spec:
       value: "med-sim-pool"
       effect: "NoSchedule"
     container:
-      image: "us.gcr.io/vcm-ml/fv3net:latest"
+      image: "us.gcr.io/vcm-ml/fv3net:{{workflow.parameters.image_tag}}"
```

Comment:
Changed when debugging the performance regression (model overwrite).  I left because it seems more consistent, but I understand that it gets annoying when testing new prog-run or training images and am fine reverting.

Reply:
I often push prog run images from my VM and submit experiments, so would prefer to revert this change...Maybe 95% of my experiments lately have been like this. I'm fine with adding more argo parameters as long as the default is "latest".